packages/rest/src/__tests__/fixtures/mixins/findByTitleInterface.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/repository
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Model} from '@loopback/repository';
+
+export interface FindByTitleInterface<M extends Model> {
+  findByTitle(title: string): Promise<M[]>;
+}
```

Comment:
Please rename this file to `find-by-title-interface.ts`.

Personally, I find it too verbose to have a dedicated file for this interface, have you considering moving the interface to the same file where `FindByTitleControllerMixin` is defined?